### ISSUE TITLE
Reformat an existing tribble() call

### ISSUE THREAD

**Issue by jennybc** [MilesMcBain/datapasta#57]
If I've got a pre-existing `tribble()` call, does this package help me to format it nicely, i.e. line up the commas?

**Comment by MilesMcBain** [comment]
Yes. You can do this:
```tibble::tribble(
  ~a, ~hot,
  ~mess,
  1, 1,
  2, 2,
  3, 3
) %>%
  datapasta::dpasta()
tibble::tribble(
  ~a, ~hot, ~mess,
   1,    1,     2,
   2,    3,     3
  )
```
`dpasta()` is a bit flaky atm but for this usecase it should work well. The issue is it doesn't handle _some_ of the types you can have in a Tibble but can't write in a Tribble.

I've been thinking about adding a feature to the addin that would do inline replacement if you have an active selection. Seems like that would work really well for this usecase. Any thoughts on that?

**Comment by jennybc** [comment]
> inline replacement if you have an active selection

Yes that is along the lines of what I was hoping for. I'm so used to being able to select code and reflow or reformat it. When I'm typing spaces to make a `tribble()` call look good, I keep thinking that the computer  should be doing this!

Your suggestion to pipe the current tribble call to `data pasta::dpasta()` worked for me. Part of what's tricky is RStudio's eagerness to reformat R code upon paste. So the fact that you're using functions from rstudioapi to insert the text is helpful. Even once you have the call formatted the way you want, all is takes is one copy/paste and the leading whitespace is mangled again.

**Comment by Minh-AnhHuynh** [comment]
Unless I'm missing something, I would like to add that the Quarto option can appropriately format citations (@citation) but can't do short captions, while kable can do caption.short but fails to cite... there's no win there. I'll have to hard code my citation in the mean time. This is for tables specifically.
